tethysext/atcore/models/resource_workflow_steps/dataset_input_rws.py

```diff
@@ -0,0 +1,53 @@
+"""
+********************************************************************************
+* Name: form_input_rws.py
+* Author: glarsen, mlebaron
+* Created On: October 17, 2019
+* Copyright: (c) Aquaveo 2019
+********************************************************************************
+"""
+from tethysext.atcore.models.app_users import ResourceWorkflowStep
+
+
+class FormInputRWS(ResourceWorkflowStep):
```

Comment:
We already have a "FormInputRWS" class.... Rename class to:   DatasetInputRWS.

Reply:
Updated the name of the class.